src/treeffuser/_base_tabular_diffusion.py

```diff
@@ -188,19 +274,26 @@ def _score_fn(y, t):
 
     def predict(
         self,
-        X: Float[ndarray, "batch x_dim"],
+        X: ndarray,
```

Comment:
Same comment about not getting rid of type annotations and simply clarifying behavior in docstring